content/api/trafiklab-apis/netex-regional/siri.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Siri
+aliases:
+- "api/netex-regional-siri"
+weight: 20 
+date: 2022-04-14
+---
+
+Siri is service Interface for Real Time Information.
```

Comment:
This can be expanded a bit:

Service Interface for Real-time Information (SIRI) is a data-format for real-time data. This data contains updates and real-time changes to the static NeTEx data. The two data formats should be used together, since  the real-time data refers to lines, ServiceJourneys, ... defined in the static data.